### PR TITLE
Fixes broken regions and phys location UI tests

### PR DESCRIPTION
**Pull request by mitchell852**
## What does this PR (Pull Request) do?

- [x] This PR is not related to any Issue 

other UI tests (regions, phys locations) depend on a division being available. deleting the newly created division causes the other tests to fail. This PR does not require additional tests or documentation.

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
Run the [UI tests](https://github.com/apache/trafficcontrol/tree/master/traffic_portal/test/end_to_end)

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: traffic_portal/test/end_to_end/divisions/divisions-spec.js [hunk 1]
```diff
@@ -67,11 +67,4 @@ describe('Traffic Portal Divisions Test Suite', function() {
 		expect(pageData.name.getText() === myNewDiv.name + ' updated');
 	});
 
-	it('should delete the new division', function() {
-		console.log('Deleting the new division: ' + myNewDiv.name + ' updated');
-		pageData.deleteButton.click();
-		pageData.confirmWithNameInput.sendKeys(myNewDiv.name + ' updated');
```
Comment (ocket8888) [inline traffic_portal/test/end_to_end/divisions/divisions-spec.js]: Are you sure the problem isn't the `+ ' updated'` here?

Comment (mitchell852) [reply traffic_portal/test/end_to_end/divisions/divisions-spec.js]: no, the delete works fine. the problem is that for these tests to work,  you can't delete certain things. 

i.e. a division is needed to create a region which is needed to create a phys location. therefore, divisions-spec and regions-spec can't have a 'delete' test.

Comment (mitchell852) [reply traffic_portal/test/end_to_end/divisions/divisions-spec.js]: i will create another PR that does all the resource cleanup (deleting) at the end of all the tests.

Comment (ocket8888) [reply traffic_portal/test/end_to_end/divisions/divisions-spec.js]: ah, okay

**Pull request merged**